### DISCUSSION TITLE
ComboBox not scaling helper text

### DISCUSSION THREAD

**Discussion by mouawadma** [MaterialDesignInXAML/MaterialDesignInXamlToolkit#3151] [Q&A] [answered]
Hello, I tried to implement the example combobox with scaling helper text:

```
<ComboBox MinWidth="72"
                  materialDesign:HintAssist.FloatingOffset="0, -24"
                  materialDesign:HintAssist.FloatingScale="1.5"
                  materialDesign:HintAssist.Hint="(large float hint)"
                  ItemsSource="{Binding LongIntegerList}"
                  Style="{StaticResource MaterialDesignFloatingHintComboBox}" />
```

You can see that it has not scaled the helper text. Any ideas why?

![image](https://user-images.githubusercontent.com/94063112/228394771-ac3f1973-67c1-47e5-99b3-c378af57bee2.png)

**Comment by nicolaihenriksen** [comment, marked as answer]
@mouawadma the scaling is only applied when the hint is floated. For that to happen with the `ComboBox`, I believe  you also need to make it editable.
